lib/testmap.py

```diff
@@ -178,47 +178,40 @@ def contexts(image: str, *scenarios: Iterable[str], repo: str | None = None) ->
     },
     'candlepin/subscription-manager': {
         'main': [
-            'rhel-9-0',
-            'rhel-9-2',
-            'rhel-9-3',
-            'rhel-9-4',
+            'rhel-9-5',
             'fedora-39',
             'fedora-40',
         ],
         'subscription-manager-1.28': [
-            'rhel-8-4',
-            'rhel-8-6',
             'rhel-8-8',
-            'rhel-8-9',
-        ],
-        'subscription-manager-1.28.29': [
-            'rhel-8-6',
+            'rhel-8-10',
         ],
         'subscription-manager-1.28.36': [
             'rhel-8-8',
         ],
-        'subscription-manager-1.29.26': [
-            'rhel-9-0',
+        'subscription-manager-1.29': [
+            'rhel-9-2',
+            'rhel-9-4',
+            'rhel-9-5',
         ],
         'subscription-manager-1.29.33': [
             'rhel-9-2',
         ],
+        'subscription-manager-1.29.40': [
+            'rhel-9-4',
+        ],
         '_manual': [
-            'rhel-8-10',
-            'rhel-9-5',
             'rhel-10-0',
         ],
     },
     'candlepin/subscription-manager-cockpit': {
         'main': [
-            'centos-9-stream',
-            'rhel-9-3',
-            'rhel-9-4',
+            'centos-9-stream/subscription-manager-1.29',
+            'rhel-9-5/subscription-manager-1.29',
```

Comment:
Is this correct?

Reply:
yes, the 1.29 branch targets rhel 9.x -- see also the branch map above for sub-man

Reply:
But that is different, it lists the branches under the branch name. This would require to trigger:

```
centos-9-stream/subscription-manager-1.29@candlepin/subscription-manager-cockpit
```
? Doesn't seem to make sense compared to the previous triggers

Reply:
sub-man-cockpit implements the "context" as branch of upstream sub-man to use for testing, defaulting to "main"; the value "system" means to use whatever sub-man is available on the system

Reply:
I triggered it but have a hard time wondering if it will work. The other changes look more trivial :)

Reply:
it worked, as expected:
```
git -C subscription-manager fetch --quiet --depth=1 origin subscription-manager-1.29
git -C subscription-manager checkout --quiet FETCH_HEAD
```

if you look at the history of the mappings for sub-man-cockpit, there used to contexts like this in the past